templates/expo-template-blank-typescript/app.json

```diff
@@ -11,17 +11,13 @@
       "resizeMode": "contain",
       "backgroundColor": "#ffffff"
     },
-    "updates": {
-      "fallbackToCacheTimeout": 0
-    },
-    "assetBundlePatterns": ["**/*"],
```

Comment:
I believe the only use of `assetBundlePatterns` besides updates was in classic builds, which are now sunsetted. I think these changes are ok, provided that `eas update:configure` adds the properties back to the config.

Reply:
Sadly, it appears that `eas update:configure` does not currently add these properties if they are missing, so that would have to be fixed before we can make this change.

Reply:
It seems that the default for this value is 0 anyways, so it's okay for this to be omitted. Though omitting it may cause it to be harder to figure out how to configure it for developers, but I guess maybe that's either a docs problem or we should add it to `eas update:configure` (currently that command just sets `updates.url`).